src/api-explorer/v4-0/HotelService.swagger2.json

```
@@ -2177,6 +2180,28 @@
       ],
       "type": "object"
     },
+    "ReservationCriteria_hrefs": {
+      "description": "Collection of URLs related to the booking, such as the Change Notification callback endpoint.",
+      "type": "object",
+      "properties": {
+        "changeNotificationV4": {
+          "type": "string",
+          "description": "Callback URL for the Change Notification API. The base URI varies depending on the datacenter where the booking is processed. The {TRIP_ID} should be replaced by the actual Trip ID.",
+          "example": "https://us2.api.concursolutions.com/travel/v4/trips/{TRIP_ID}/hotels/change-notification",
```

I dont think its needed to add all datacenters example for the suppliers, our documentation does not state those anywhere.
```suggestion
          "example": "https://{DATACENTER}.api.concursolutions.com/travel/v4/trips/{TRIP_ID}/hotels/change-notification",
```